specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,408 @@
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkDataList": {
+      "type": "object",
+      "description": "List of network data",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkData"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "NetworkData": {
+      "type": "object",
+      "description": "Network data on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkDataProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "NetworkDataProperties": {
+      "type": "object",
+      "description": "Describes the properties of the network data.",
+      "properties": {
+        "connectableResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkDataConnectableResource"
+          }
+        }
+      }
+    },
+    "NetworkDataConnectableResource": {
+      "type": "object",
+      "description": "Describes the allowed inbound and outbound traffic of an Azure resource",
+      "properties": {
+        "connectedResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connectable resource id"
+        },
+        "connectionType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The connection type"
+        },
+        "allowedConnections": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The allowed connections details",
+          "properties": {
+            "calculatedDateTime": {
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true,
+              "description": "The UTC time on which the allowed connections was calculated"
+            },
+            "inboundConnectedResources": {
+              "readOnly": true,
+              "type": "object",
+              "description": "The allowed connections inbound details",
+              "properties": {
+                "tcpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound tcp ports summary"
+                },
+                "udpPortsSummary": {
+                  "readOnly": true,
+                  "type": "integer",
+                  "description": "The allowed connections inbound udp ports summary"
+                },
+                "tcpPorts": {
+                  "readOnly": true,
+                  "type": "string",
```

Comment:
why is it not an array of int?

Reply:
saw the example :)
array of string, and also specify in the comments that it can be singular or range the examples of the format